src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -102,6 +102,7 @@
   private int maxRetries;
   private long retryBackoffMs;
   private final Time time = new SystemTime();
+  private int retryOnConflict;
```

Comment:
can be `final` I suppose

Reply:
final would require us to initialize it in the other 2-3 constructors :/

Reply:
Adding the statements should not be an issue. The variable is initialized implicitly as `false` but it's not final. Member fields that are immutable outside the constructor should be `final`.